src/Styleguide/Artwork/Banner.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import styled from "styled-components"
+import { Col, Row } from "../Elements/Grid"
+
+const Flex = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+
+interface RoundedImageProps {
+  size: string
+}
+
+const RoundedImage = styled.img.attrs<RoundedImageProps>({})`
+  width: ${(props: RoundedImageProps) => props.size};
```

Comment:
This isn't needed now that `styled.img.attrs<RoundedImageProps>` is there, right?